src/Chunk.ts

```diff
@@ -591,15 +591,27 @@ export default class Chunk {
 	}
 
 	private getChunkDependencyDeclarations(): ChunkDependencies {
+		// if an entry point facade, inline the execution list to avoid loading latency
+		if (this.isEntryModuleFacade) {
+			const inlineDeepChunkDependencies = (dep: Chunk | ExternalModule) => {
+				if (dep === this || this.dependencies.indexOf(dep) !== -1) return;
+				this.dependencies.push(dep);
+				if (dep instanceof Chunk) dep.dependencies.forEach(inlineDeepChunkDependencies);
+			};
+			this.dependencies.forEach(dep => {
+				if (dep instanceof Chunk) dep.dependencies.forEach(inlineDeepChunkDependencies);
+			});
+		} else {
+			// shortcut cross-chunk relations can be added by traceExport
+			this.imports.forEach(impt => {
+				if (this.dependencies.indexOf(impt.module) === -1) this.dependencies.push(impt.module);
+			});
+		}
```

Comment:
As this `if-else` is basically a preparation step for `getChunkDependencyDeclarations`  with the sole purpose of adding some additional dependencies to `this.dependencies`, I would suggest extracting it into a separate private method, e.g. `addAdditionalDependencies` or `addDerivedDependencies` or whatever rings best with you.
Then I would suggest to call this method not from within `getChunkDependencyDeclarations` but directly from `getModuleDeclarations`. Thus it is immediately visible that this public method has a side-effect on a chunk's dependencies (which may or may not be helpful for future refinements)